packages/typescript/internal/ts_project.bzl

```diff
@@ -610,10 +614,14 @@ def ts_project_macro(
     """
 
     if srcs == None:
+        include = ["**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx"]
```

Comment:
something is wrong with the logic here, you can never leave this block without including .js and .jsx where previously you could

Reply:
good spot, thanks. i feel that internal details like this would be easier to catch with unit tests, so we don't have to figure out ways to make everything express through mini-integration tests (which is hard for this one, for instance). but i guess the tooling is not in place for this?

Reply:
`generated_file_tests` for generated tsconfigs would have caught this. that's not a substitute for testing some of these internal pieces separately, but would be an improvement on what's there today. perhaps i can look at that later on in a different pr.

Reply:
yes sometimes the "mini-integration test" is not the best choice. There's a way to assert on what actions are produced by a rule implementation but not run them (like running `bazel aquery` and asserting on the result" which I'd love to try.

maybe for this even a smaller test fixture using the testing libraries in skylib would work, but I haven't investigated those for a while.